inc/userinjection.class.php

```diff
@@ -55,7 +55,95 @@ public function connectedTo()
 
     public function isNullable($field)
     {
-        return true; // By default, all fields can be null
+        return in_array($field, [
```

Comment:
Please list them in alphabetical order, it will be easier to maintain for future additions/deletions.

Reply:
I did it that way to respect the code already written in this PR : 

https://github.com/pluginsGLPI/datainjection/pull/502/files#diff-ed87f9bda1f73e2d962ded1532617a242b44027fa95b5cef2f03012ff3432eba

as there was already an example in this file : 

https://github.com/pluginsGLPI/datainjection/pull/502/files#diff-91a615ac23201a0394dd8c6b70359227cc7799abeedb7822463b3d46b8a9822c